src/main/resources/api/service_openapi.yaml

```diff
@@ -424,6 +432,20 @@ components:
           type: string
           format: uuid
 
+    CloningInstructionsEnum:
+      type: string
+      description: Instructions for copying this reference when cloning the workspace
+      enum: ['COPY_NOTHING', 'COPY_DEFINITION', 'COPY_RESOURCE', 'COPY_REFERENCE']
+
+    ReferenceTypeEnum:
+      type: string
+      description: The type of a reference, if an uncontrolled resource
+      # TODO: swagger-codegen always looks for and removes a common prefix for
+      # enum values, which breaks enums that only have a single value.
+      # PLACEHOLDER_VALUE should be removed when we have an actual second value
+      # to use.
+      enum: ['DATA_REPO_SNAPSHOT', 'PLACEHOLDER_VALUE'] #eventually include GCS bucket, etc.
+
```

Comment:
oof. Is there any other solution to this? (fine to do in a follow-on PR)

Reply:
Yeah... 

As far as I can tell, there's no option to disable this behavior in swagger-codegen (there is back in OpenApiGenerator, but we've already had one migration...). 
We could change the value to `DATAREPOSNAPSHOT`, but the lack of underscores is more difficult to read.
The old value of `DataRepoSnapshot` caused issues with enum parsing, since by default swagger-codegen enums and their string values don't always match

Reply:
This comment about using a custom generator is enticing: https://github.com/swagger-api/swagger-codegen-generators/issues/246#issuecomment-442629121 but it feels like overkill to use a custom generator just for this.